README.md

```diff
@@ -1,2 +1,57 @@
-# darwinia-common
-[Substrate](https://github.com/paritytech/substrate)-like common libraries, but only for Darwinia blockchains!
+# Darwinia Runtime Modules Library
+The Darwinia Runtime Modules Library (DRML) is a darwinia.network maintained collection of Substrate runtime modules.
+
+## Runtime Modules Overview
+- [darwinia-balances](./frame/balances)
+	- Desc.
```

Comment:
I think we do not need this `Desc`. It is better to add a readme document under pallet directory if we want to add pallet description.

Reply:
Follow this https://github.com/open-web3-stack/open-runtime-module-library

But TODO

Reply:
See, I recommend adding a one-line description in this pull request.